examples/cpp/Breakpoint.cc

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
Could you add a license header to all new files you added? You can look at other files for examples.

Reply:
Yeah, but I wasn't sure because I saw in all other examples, copyrights were given to some company. How to do this if you are uploading the code individually?

Adding just this following line would do?
Licensed under the Apache License, Version 2.0 (the "License")

Reply:
The following
```
Licensed under the Apache License, Version 2.0 (the "License")
```
should be okay.